index.js

```diff
@@ -37,13 +38,19 @@ module.exports = {
         ' having trouble registering itself to the parent application.');
     }
 
-    var pathBase = this.project.addonPackages['@html-next/flexi-default-styles'].path;
-    compileScssVariables(path.join(pathBase, 'addon/styles'), this.flexiConfig());
-
     this.app = app;
     return app;
   },
 
+  treeForAddonStyles(tree) {
+    console.log('OMG')
+    let stylesRoot = path.join(__dirname, 'addon/styles');
+    return mergeTrees([
+      stylesRoot,
```

Comment:
@stefanpenner Is there a particular reason you went with `path.join(__dirname, 'addon/styles')` instead of just using `tree`?